docs/example/bootstrap.sh

```diff
@@ -1,52 +1,130 @@
 #!/bin/bash
 
 # Script for manually testing the bootstrap process using a container
+catch() {
+if [ ! -f "$CONSTRAINTS" ] && [ -f "$CONSTRAINTS_FILE" ]; then
+	rm -f "$CONSTRAINTS_FILE"
+fi
+
+if [ ! -f "$REQUIREMENTS" ] && [ -f "$REQUIREMENTS_FILE" ]; then
+	rm -f "$REQUIREMENTS_FILE"
+fi
+}
+trap 'catch' EXIT INT
+
+usage() {
+	echo "Usage: [-f <fromager arguments> | -k <seconds>] CONTAINERFILE CONSTRAINTS REQUIREMENTS"
+	echo "       -c: Execute different command in container (must be passed with double quotes)"
+	echo "       -f: additional fromager arguments"
+	echo "       -h: help (this message)"
+	echo "       -k: set number of seconds to keep container running after execution"
+}
+
+COMMAND=""
+FROMAGER_ARGS=""
+KEEPALIVE=0
+
+BASE_ARGS=()
+while [[ $# -gt 0 ]]; do
+	case $1 in
+	-h)
+		usage
+		exit 0
+		;;
+	-c)
+		COMMAND="$2"
+		shift
+		shift
+		;;
+	-f)
+		FROMAGER_ARGS="$2"
+		shift
+		shift
+		;;
+	-k)
+		KEEPALIVE="$2"
+		re='^[0-9]+$'
+		if ! [[ "$KEEPALIVE" =~ $re ]]; then
+			echo "-k value must be a number of seconds to keep container running"
+			exit 1
+		fi
+		shift
+		shift
+		;;
+	*)
+	BASE_ARGS+=("$1")
+	shift
+	;;
+	esac
+done
 
-usage() { echo "Usage: CONSTRAINTS_FILE REQUIREMENTS_FILE" 1>&2; exit 1; }
+# reset the args with base arguments
+set -- "${BASE_ARGS[@]}"
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -lt 3 ]; then
    usage
+   exit 1
 fi
 
 set -x
 set -e
 set -o pipefail
 
-CONSTRAINTS_FILE="$1"
-REQUIREMENTS_FILE="$2"
+CONTAINERFILE="$1"
+CONSTRAINTS="$2"
+REQUIREMENTS="$3"
+
+if [ ! -f "$CONSTRAINTS" ]; then
+	CONSTRAINTS_FILE=$(mktemp)
+	echo "$CONSTRAINTS"  | tr ',' '\n' > "$CONSTRAINTS_FILE"
+else
+	CONSTRAINTS_FILE="./$CONSTRAINTS"
+fi
+
+if [ ! -f "$REQUIREMENTS" ]; then
+	REQUIREMENTS_FILE=$(mktemp)
+	echo "$REQUIREMENTS"  | tr ',' '\n' > "$REQUIREMENTS_FILE"
+else
+	REQUIREMENTS_FILE="./$REQUIREMENTS"
+fi
 
-CONTAINERFILE="Containerfile"
 IMAGE="wheels-builder"
+TOOL_SERVER_URL=https://pyai.fedorainfracloud.org/internal/tools/+simple/
```

Comment:
Unless you've updated it, this package index is WAY out of date. Do we need it?

Reply:
Nope, we don't.  I've dropped this change.